openmaptiles/sql.py

```diff
@@ -117,6 +119,29 @@ def get_slice_language_tags(tileset):
 $$ LANGUAGE SQL IMMUTABLE;
 """
 
+def create_fnc_is_table_existing():
```

Comment:
we probably should not create global functions that are only needed during the import